File: c/15.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    srand(time(0));
    int randomNum = (rand() % 20) + 1; // random system 10/10

    // the 
    if (randomNum > 10) {
        printf("The random number is %d and is greater than 10\n", randomNum);
    } else if (randomNum < 10) {
        printf("The random number is %d and is less than 10\n", randomNum);
    }

    return 0;
}

File: kotlin/15.kt
fun main(){
    val randNum : Int = (Math.random()*20).toInt()+1;
    if(randNum > 10){
        println("The random number is ${randNum} and is greater than 10");
    } else {
        println("The random number is ${randNum} and is less than 10");
    }
}